skorch/net.py

```diff
@@ -1792,7 +1792,10 @@ def get_params(self, deep=True, **kwargs):
         # special treatment.
         params_cb = self._get_params_callbacks(deep=deep)
         params.update(params_cb)
-        return params
+
+        # don't include the following attributes
+        to_exclude = {'_modules', '_criteria', '_optimizers'}
+        return {key: val for key, val in params.items() if key not in to_exclude}
```

Comment:
On `master`, these attributes are not included in `get_params`.

Reply:
What do you mean? On current master, after merging #751, `get_params` includes those.

Reply:
Ah I was mistaken. (I was on a older version of master locally)

Can we add a test here:

https://github.com/skorch-dev/skorch/blob/812f54d76cf6c7a1aba213dc2290a47607c6f55c/skorch/tests/test_net.py#L1409

Something like:

```python
assert `_modules` not in params
```

Reply:
Good idea. I added a separate test below because that one was already big enough.